script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -604,63 +652,66 @@ I hear a woman's voice coming from the other side of the door.
 [sha:ff1dbfda07574698403d86d24e4d297ee54d867e]{
 -- Page 10012, Offset 33186.
 -- ……答える余裕がない。
-...I have no leeway to answer back.
+I don't have the presence of mind to answer her.
 }
 [sha:6e4fe9935e6dd2732af1766503a8bea9fb919feb]{
 -- Page 10012, Offset 33187.
 -- コトバ。自分のコトバを、思い出せない。
-Language. I can't even remember my own language.
+Words... I can't even remember how to speak.
 }
 [sha:df08e0d89cade43b45a4fe28a801f8597f3116d6]{
 -- Page 33, Offset 33188.
 -- 頭に浮かぶのは。
-The only thing that's coming to mind is...
+My mind can only picture one thing―――
 }
 [sha:d2734b7a893e327d8c43f356d19f5115dc24cb52]{
 -- Page 33, Offset 33189.
 -- 頭に、浮かぶのは。
-The only... thing that's coming to mind is...
+My mind can only picture... one thing―――
 }
 [sha:85f280371538dcbf9e7839d7a09d172bc86d2ec7]{
 -- Page 10014, Offset 33190.
 -- こみ上げる衝動のままに。
 // reordered for flow
-The sight of myself, with my throat crying out―――
+The sight of myself―――
 }
 [sha:1c8b26b22185b48a71befe794e82eae3108eab0c]{
 -- Page 10016, Offset 33191.
 -- 生き物を、■したくて、■したくて
-wanting to ■■■ and ■■■―――
+// Interpreting 鳴らす here as his throat 'banging the drum' of killing/drinking whatever.
+// Not doing a more direct TL for fear of it coming across like he's actually yelling.
+Throat burning with the desire to ■■■ and ■■■―――
 }
 [sha:0c2886587d22cf846d68649e2ef7d73181691290]{
 -- Page 10016, Offset 33192.
 -- 喉を鳴らしている、自分の姿だけ。
-as I yield to my surging impulses.
+As I yield to my surging impulses.
 }
 [sha:402a2ccc928c8f611faabb3a21b807d9ab3e0d24]{
 -- Page 34, Offset 33193.
 -- 「うわあああ――――！！！！！！！」
-"UWAAAAAH――――!!!!!!"
+"%{i}UWAAAAAH%{/i}――――!!!!!!"
 }
 [sha:a9450f48b98cee8230915d748932758269cdd089]{
 -- Page 34, Offset 33194.
 -- 頭。頭を壁に打ち付ける。
-My head. I bang my head against the wall.
+// Feels more natural to use the target here in EN for the repetition.
+The wall. I bang my head against the wall.
 }
 [sha:42a7d03698132cc299e8a8ac50a2d9054e8f2e19]{
 -- Page 34, Offset 33195.
 -- 何度も何度も、額が割れそうになるまで打ちつける。
-Over and over again, I bang my head against the wall until my forehead is about to split.
+I smash it into the wall over and over again until my forehead is about to crack.
 }
 [sha:2a1edee8b9d0322769c906e73d88950325295f64]{
 -- Page 34, Offset 33196.
 -- それでも―――■■をズタズタにするイメージが、頭の中から消えてくれない。
-But still―――the image of shredding ■■ into pieces won't leave my head.
+But try as I might――I can't dislodge the image of shredding ■■■ into pieces from my mind.
 }
 [sha:335e42bae910325582c78e7cf8831d3c56f6588b]{
 -- Page 35, Offset 33197.
 -- ……そういう、コトか。
-...So that's how it is.
+So that's how it is...
 }
 [sha:e3be751749701df3423480b5dbb1dafbe39731a1]{
 -- Page 35, Offset 33198.
```

Comment:
We use urge for most of the stuff regarding vampirism, so that's probably best here?